pom.xml

```diff
@@ -21,7 +21,10 @@
         <jacoco.version>0.8.7</jacoco.version>
         <sonar.language>java</sonar.language>
         <sonar.java.coveragePlugin>jacoco</sonar.java.coveragePlugin>
-        <sonar.coverage.exclusions>**/configurations/*</sonar.coverage.exclusions>
+        <sonar.coverage.exclusions>
+            **/configurations/*,
+            **/kafka/KafkaDbTransactionSynchronizer.java
```

Comment:
Why have we excluded this from code coverage?

Reply:
KafkaEntityListenerIT does cover this with tests. But Sonar was reporting the lines weren't covered.

Reply:
Yeh so that is reporting Method - 100% and line - 88%

Reply:
This is now fixed. We needed to put the test class in the exact same package as the corresponding main code

Reply:
@stephenkaye-ho please resolve comment if you're happy